### DISCUSSION TITLE
Tasmota AP selection

### DISCUSSION THREAD

**Discussion by PabloVogel** [arendst/Tasmota#10225] [Q&A] [answered]
I'm using fw 9.2.0 with core 2.7.4.9 and I have a question about multiple APs.I have 3 AP with the same SSID and password, a setup known as roaming, so the clients are free to bind to the best signal.
But the sonoff devices seem to always bind to the same access point, regardless of the signal  as long it's reachable.
Example: if I prime a new  sonoff near AP1, then relocate near AP2 it always use AP1 unless I shutoff AP1, and then will always use AP2 no matter what.
Are we supposed to have multiple SSID and manually balance devices using AP1 and AP2 config?
What are your reccomendations for a multiple AP setup?

Regards

**Comment by sfromis** [comment, marked as answer]
You may want to look into SetOption56 and SetOption57 for dynamic AP selection. Works in my setup with 3 same-named AP.
